### PR TITLE
Add logging to show counter writing

### PR DESCRIPTION
**Pull request by daniele-moro**
We still don't support counter writing, but PFCP-agent will start issuing counter write.
For debugging, we logs those write request, without doing any action on the data plane, while replying positively to pfcp-agent.

### CODE REVIEW

**Comment by daniele-moro** [comment]
Need: https://github.com/omec-project/upf/pull/607 to be merged first and to change the reference in the `.env.stable`.

**Comment by daniele-moro** [comment]
retest this please

File: .env.stable [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #
 
 
-PFCP_AGENT_IMAGE=omecproject/upf-epc-pfcpiface:master-2f04e3a
```
Comment (pierventre) [inline .env.stable:6]: do we have to revert this before merging ?

**Review by pierventre — APPROVED** [review]

File: .env.stable (hunk 1 above)
Comment (daniele-moro) [reply .env.stable:6]: Yes, we need first to merge the change in pfcp-agent.

**Review by pierventre — APPROVED** [review]

File: .env.stable (hunk 1 above)
Comment (pierventre) [reply .env.stable:6]: All the prs have been merged in upf. Should we revert this ?

**Review by pierventre — APPROVED** [review]
Please remove the label and fix the conflicts

**Pull request merged**